gandalf.js

```diff
@@ -1,30 +1,29 @@
 /**
```

Comment:
SCRIPT-Consider extracting this repeated logic into a separate function to avoid duplication.